README.md

```diff
@@ -2,7 +2,9 @@
 
 ---
 
-:spider_web: **Archived**: This repo is no longer maintained and is archived. Please use the following public module: https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances
+## :spider_web: **Archived**: This repo is no longer maintained and is archived. Please use the following public module if consuming from github: [terraform-ibm-observability-instances](https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances)
+
+## :warning: This module will be deleted from the Hashicorp registry on the 1st August 2023. If consuming from Hashicorp, please use [observability-instances](https://registry.terraform.io/modules/terraform-ibm-modules/observability-instances/ibm/latest) instead.
```

Comment:
```suggestion
## Module archived

:spider_web: This repo is no longer maintained and is archived. Use the following public module if you consume the module from GitHub: [terraform-ibm-observability-instances](https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances)


## Changes if you use the Hashicorp registry

:exclamation: **Important:**  This module will be deleted from the Hashicorp registry on 1 August 2023. 

If you  consume the module from the Hashicorp registry, use [observability-instances](https://registry.terraform.io/modules/terraform-ibm-modules/observability-instances/ibm/latest) instead.
```

How about this. It preserves the heading as a indicator of the section but still calls it out. We also don't use `warning` unless bodily harm might occur.